inventory/src/pages/kitLoanReturn.js

```diff
@@ -0,0 +1,183 @@
+import { Box, Fab, Divider, useTheme, useMediaQuery } from '@mui/material';
+import React, { useState } from 'react';
+
+import { NavBar, Footer, LoadingSpinner } from '../components';
+import {
+  KitInfoSection,
+  KitItemReturnSection,
+  ConfirmationModal,
+} from '../components';
+import { useUser, useKit, useKitRecipe } from '../hooks/queries';
+
+export const KitLoanReturn = () => {
+  const { data: userData } = useUser();
+
+  const params = new URLSearchParams(window.location.search);
+  const kitId = params.get('kitId');
+  const { data: kitData } = useKit({ kitId: kitId });
```

Comment:
can we add some logic to render an error message when the kit is not currently loaned